src/iterateJsdoc.js

```diff
@@ -870,7 +871,7 @@ const getUtils = (
   };
 
   utils.filterTags = (filter) => {
-    return jsdocUtils.filterTags(jsdoc.tags, filter);
+    return jsdocUtils.filterTags(jsdocUtils.getAllTags(jsdoc), filter);
```

Comment:
The `filterTags` utility is used by a number of other rules. Have you confirmed that it can work in those other contexts? I think we probably need to revisit the rules on a case-by-case basis, also ensuring they are polymorphic enough in structure and fixing rules to be viable for retrieval without further code modifications.

Reply:
Good point 👍 The other rules consider "tags" as non-inline, so it might break if these are included. I could introduce a second optional `includeInlineTags` argument (optionally `false`) or introduce another function called `filterAllTags` or make two calls, the original `filterTags` and another `filterInlineTags`? What do you think?

Reply:
An additional `includeInlineTags` argument SGTM but ok if the code demands something else...